magento/magento-cloud-docker
# Disable blackfire PHP extension by default

### Description

At the moment **blackfire** PHP extension can not be switched off, since it is enabled not by **PHP_EXTENSIONS** ENV variable. You can check it by executing command below:

- `docker run --rm magento/magento-cloud-docker-php:7.4-cli-1.2.0 php -v`

Note the last message:

> with blackfire v1.43.0~linux-x64-non_zts74, https://blackfire.io, by Blackfire

Even if monitoring is not configured - it affects performance. Just compare load time for listing/product page with enabled/disabled blackfire extension after execution of "**setup:upgrade**" / "**cache:flush**" commands.

### Manual testing scenarios
1. Blackfire should not be enabled by default. Execute and verify output (here should be no any mentions about blackfire).
`docker run --rm magento/magento-cloud-docker-php:7.4-cli-1.2.0 php -v`

2. If you want to enable it - just add "**blackfire**" to the list of extensions via **PHP_EXTENSIONS** env variable:
`docker run --rm -e "PHP_EXTENSIONS=blackfire" magento/magento-cloud-docker-php:7.4-cli-1.2.0 php -v`

Inspect php version output. Now you should see that blackfire is enabled. Here should be no warning like that:

> warning: blackfire (blackfire) is already loaded!

### Contribution checklist
